docs/guides/admin/docs/modules/notification.md

```diff
@@ -0,0 +1,37 @@
+# Mattermost Notification
+#### Example for mattermost-notify operation:
```

Comment:
You shouldn't skip a header level. It should be something like:

```
Header Level One
==============

Header Level Two
------------------------
```

…or it's equivalent using `#` and `##`